checker/checkFlags.ml

```diff
@@ -11,13 +11,19 @@
 open Declarations
 
 let set_local_flags flags env =
+  (* Explicitly ignored flags have been commented out *)
```

Comment:
until we add some in the datatype

Reply:
Right, so we should perhaps do this a better way.

Reply:
Done.